packages/lib/response/service.ts

```diff
@@ -392,19 +395,76 @@ export const getResponse = async (responseId: string): Promise<TResponse | null>
     : null;
 };
 
+export const getAttributesFromResponses = async (surveyId: string): Promise<TSurveyPersonAttributes> => {
```

Comment:
We might also here get an issue with the response size from Prisma for a very large amount of users. I think maybe in the future we might only pull the list of the current attributeClasses for the filtering and let the user insert a value as free text. But for now I think it's a good solution and let's go with it as far as we can 😊

Reply:
Yes, exactly. I will think of possible optimisations for it.